src/components/AuditLogsTable/hooks.ts

```diff
@@ -23,14 +24,12 @@ export const useFetchAuditEvents = () => {
   const [queryParams] = useQueryParams<
     {
       page: string;
-      limit: string;
     } & AuditLogFilters
   >({
     page: DEFAULT_PAGE,
-    limit: DEFAULT_LIMIT_VALUE.toString(),
     ...DEFAULT_AUDIT_LOG_FILTERS,
   });
-  const limit = Number(queryParams.limit);
+  const limit = Number(useSelector(getAuditEventsLimit));
```

Comment:
It looks like`getAuditEventsLimit` will return a number, so no need to wrap this in `Number(...)`